app/code/Magento/Newsletter/Controller/Manage/Save.php

```diff
@@ -65,9 +66,9 @@ public function __construct(
     /**
      * Save newsletter subscription preference action
      *
-     * @return void|null
+     * @return \Magento\Framework\App\ResponseInterface
      */
-    public function execute()
+    public function execute(): \Magento\Framework\App\ResponseInterface
```

Comment:
Unfortunately, we can't change public methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
@sidolov fixed.